main.tf

```diff
@@ -214,14 +215,17 @@ data "aws_iam_policy_document" "base" {
   }
 
   statement {
-    sid    = "AllowPushPull"
+    sid    = "AllowPushPullListDelete"
```

Comment:
This data source just outputs a `JSON` block that is passed in verbatim to the IAM policy, i.e. it will completely replace the JSON that is already there 👍🏼